product_packaging_container_deposit/models/container_deposit_order_mixin.py

```diff
@@ -108,3 +109,11 @@ def copy(self, default=None):
         return super(
             OrderMixin, self.with_context(skip_update_container_deposit=True)
         ).copy(default=default)
+
+    def write(self, vals):
+        # When an order is modified, don't recompute container deposit for each line
+        res = super(
+            OrderMixin, self.with_context(skip_update_container_deposit=True)
+        ).write(vals)
+        self.update_order_container_deposit_quantity()
```

Comment:
what's the point of calling `update_order_container_deposit_quantity` if you always skip it?
Also, wouldn't be better to check if there's any update on values that might impact on deposit container lines?

Reply:
> Your second point could be a further improvement in another PR.
I would check - at least - that you are writing on `order_line`. That's easy, no?